playback/tape_cassette.py

```diff
@@ -151,6 +154,26 @@ def _match_metadata_value(match_value, recorded_value):
 
         return recorded_value == match_value
 
+    @staticmethod
+    def operator_filter(recorded_value, metadata_value):
```

Comment:
This should be a private method until proven otherwise, so should start with _

Reply:
Done